java/ql/src/experimental/Security/CWE/CWE-918/RequestForgery.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Uncontrolled data used in network request
+ * @description Sending network requests with user-controlled data allows for request forgery attacks.
+ * @kind path-problem
+ * @problem.severity error
+ * @id java/request-forgery
```

Comment:
If I am not mistaken this query only addresses server-side request forgery and not others such as cross-site request forgery. Therefore I think we should make the `@id` more precise. I'd suggest `java/server-side-request-forgery` .